test-utils/src/data/free.rs

```diff
@@ -103,13 +103,45 @@ fn to_tx_verification_data(tx_blob: impl AsRef<[u8]>) -> VerifiedTransactionInfo
     let tx = Transaction::read(&mut tx_blob.as_slice()).unwrap();
     VerifiedTransactionInformation {
         tx_weight: tx.weight(),
-        fee: tx.rct_signatures.base.fee,
+        // TODO:
+        fee: tx_fee(&tx),
```

Comment:
Description for TODO?

Reply:
Ah I fixed the TODO but left the TODO will remove :D